### PR TITLE
test(e2e-desktop): add spectron

### PR DESCRIPTION
**Pull request by mroz22**
Let's start discussion how to test desktop apps better. Logic there is [increasing](https://github.com/trezor/trezor-suite/pull/2438), updates may break the whole thing without [noticing](https://github.com/trezor/trezor-suite/pull/2502) and any unhandled exception in main process causes crash of the application which is something we want to avoid :]

### CODE REVIEW

File: packages/suite-desktop/package.json [hunk 1]
```diff
@@ -167,6 +167,8 @@
         "next-transpile-modules": "^2.3.1",
         "react": "16.13.1",
         "react-dom": "16.13.1",
+        "react-markdown": "4.3.1",
```
Comment (keraf) [inline packages/suite-desktop/package.json:168]: Not sure why the react-markdown dependency was added here. I guess it's not required by spectron?

**Comment by keraf** [comment]
I was thinking about Spectron the other day, would definitely be useful to have it in our test stack for the desktop app.

I haven't ready up too much on it yet but how do you interact with/check the content of the renderer?

**Comment by mroz22** [comment]
there is this `app.client` object that has some API for interaction with "what is on screen". But it felt rather clumsy (compared to cypress). Need to dive into it. I hope I can manage it after this fridays freeze.

File: packages/suite-desktop/package.json (hunk 1 above)
Comment (mroz22) [reply packages/suite-desktop/package.json:168]: yeah, it looks suspicious and does not make sense. I ll check it.

**Pull request closed without merging**